src/docfx/lib/JsonUtility.cs

```diff
@@ -268,6 +268,17 @@ public static void Merge(JObject container, JObject overwrite)
             container.Merge(overwrite, s_mergeSettings);
         }
 
+        /// <summary>
+        /// Merge overwrite into container
+        /// And keep the line info of properties in overwrite which are not in container
+        /// </summary>
+        public static void MergeWithLineInfo(JObject container, JObject overwrite)
+        {
+            var original = overwrite.DeepClone() as JObject;
```

Comment:
how about make it as default behavior?

Reply:
sounds good

Reply:
Actually, it depends if we need this workaround

Reply:
move this method to configLoader as a private method